modules/mesh-task/templates/consul_client_command.tpl

```diff
@@ -17,3 +21,10 @@ exec consul agent \
   -hcl 'ports { grpc = 8502 }' \
   -hcl 'advertise_reconnect_timeout = "15m"' \
   -hcl 'enable_central_service_config = true' \
+%{ if tls ~}
+  -hcl 'ca_file = "/tmp/consul-ca-cert.pem"' \
+  -hcl 'auto_encrypt = {tls = true}' \
+  -hcl "auto_encrypt = {ip_san = [\"$ECS_IPV4\"]}" \
+  -hcl 'verify_outgoing = true' \
+  -hcl 'ports {https = 8501}' \
```

Comment:
disable http?

Reply:
Resolving since we've decided to only have HTTP.